### DISCUSSION TITLE
Which F1 driver won 2021 Championship?

### DISCUSSION THREAD

**Discussion by GameExplorer** [ParanoidUser/yolo#349] [Galaxy Trivia] [answered]
Hint: #33 driver number

**Comment by ParanoidUser** [comment, marked as answer]
My F1 racing knowledge might be a bit rusty these days. The last time I saw F1 was in 2020 COVID time, and *Lewis Hamilton* (team Mercedes) won it for the 3rd or 4th year in a row. But he drives #44, if I remember correctly. The second person that came to my mind (and the most likely winner in recent years), the guy from the Red Bull team who constantly breaking pit stop records (under 2 seconds) is *Max Verstappen*. Third would be *Sebastian Vettel*, but I heard he retired.

Are you following the Saudi Arabia 2023 qualifiers? What would you bet on this year? 🏎️

**Comment by thuongtruong1009** [comment]
The 2021 Formula 1 World Championship was won by Max Verstappen, a Dutch racing driver who drives for Red Bull Racing. It was Verstappen's first World Championship win, and he won it after a thrilling season-long battle with seven-time World Champion Lewis Hamilton, who drives for Mercedes-AMG Petronas Formula One Team. Verstappen won the championship by just 3 points in the final race of the season, which was held in Abu Dhabi on December 12, 2021.

**Comment by skzidan** [comment]
Max Verstappen won his first World Championship (and the first for a Dutch driver), driving for Red Bull Racing-Honda. Lewis Hamilton, the defending champion, finished runner-up, driving for Mercedes. Mercedes won their eighth consecutive Constructors' Championship.

**Comment by peterbaccaro** [comment]
Max Verstappen won the 2021 F1 World Championship driving for Red Bull Racing-Honda.  It was his first World Championship and the first for a Dutch driver.
